PokerGame/BetPoker.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PokerGame
+{
+    internal class BetPoker
+    {
+        public int BetValue { get; set; }
+
+        public int GetThePrize(string[] images)
+        {
+            int[] cards = new int[5];
+            string[] colors = new string[5];
+            ParseImages(images, cards, colors);
+            Array.Sort(cards);
+
+            if (IsFlush(colors))
+            {
+                if (IsStraight(cards))
+                {
+                    return cards[4] == 15 ? BetValue * 500 : BetValue * 300;
+                }
+                else
+                {
+                    return BetValue * 200;
+                }
```

Comment:
remove the else
if the above expression is true, it will return cards[4] == 15 ? BetValue * 500 : BetValue * 300;
if not you can automatically return BetValue * 200;